#!/usr/bin/env python
PACKAGE="obj_map"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("costBase", int_t, 0, "cost base", 15, 1, 100)
gen.add("offset_x", double_t, 0, "offset_x", 0, -200, 200)
gen.add("offset_y", double_t, 0, "offset_y", 0, -200, 200)
gen.add("offset_z", double_t, 0, "offset_z", 0, -200, 200)
gen.add("roll", double_t, 0, "roll", 0, -3.1415926, 3.1415926)
gen.add("pitch", double_t, 0, "pitch", 0, -3.1415926, 3.1415926)
gen.add("yaw", double_t, 0, "yaw", 0, -3.1415926, 3.1415926)
gen.add("map_height", double_t, 0, "map_height", 50, 50, 500)
gen.add("map_width", double_t, 0, "map_width", 50, 50, 500)

exit(gen.generate(PACKAGE, "points2costmap", "Points2Costmap"))